frontend/util/toasts.ts

```diff
@@ -0,0 +1,20 @@
+import toast from "react-hot-toast";
+
+export async function handleWithToast(
+	promise: Promise<Response>,
+	toastID: string = "",
+	loadingMessage?: string,
+	successMessage?: string
+) {
+	if (loadingMessage) {
+		toastID = toast.loading(loadingMessage, { id: toastID });
+	}
+	const res = await promise;
```

Comment:
Could you rename this variable to match the standard?
```suggestion
	const response = await promise;
```